app/application/validations/create_account_test.go

```diff
@@ -24,6 +24,16 @@ func Test_nameIsNotEmpty(t *testing.T) {
 	})
 }
 
+func Test_cpfIsRightSIze(t *testing.T) {
```

Comment:
Acho que ficaria legal agrupar todos os testes, se voce achar que faz sentido, por "assunto"... por cpf, por exemplo... e quem sabe praticar [table based tests](https://quii.gitbook.io/learn-go-with-tests/go-fundamentals/roman-numerals#and-the-rest).

Apenas uma sugestao pro futuro.

Reply:
Ah sim, pretendo depois montar um table test para todos, é que esses eu ainda não mexi nos testes